src/lib/sw.js

```diff
@@ -89,26 +106,12 @@ const assetExpirationPlugin = new ExpirationPlugin({
   maxEntries: 100, // allow a large number of images, but expire quickly
 });
 
-// Cache the Google Fonts stylesheets with a stale-while-revalidate strategy.
-workboxRouting.registerRoute(
-  /^https:\/\/fonts\.googleapis\.com/,
-  new workboxStrategies.StaleWhileRevalidate({
-    cacheName: 'google-fonts-stylesheets',
-    plugins: [externalExpirationPlugin],
-  }),
-);
-
 // Cache the underlying font files with a cache-first strategy for 1 year.
 workboxRouting.registerRoute(
-  /^https:\/\/fonts\.gstatic\.com/,
+  ({request}) => request.destination === 'font',
```

Comment:
nice, it's the future!

There's a small set of old browsers which don't support this (but which otherwise will get our SW—like Chrome 61-64). I don't think it's worth worrying about them.

Reply:
👍